### ISSUE TITLE
Addition of Starter Ship Template

### ISSUE THREAD

**Issue by idfreud** [Eggs-D-Studios/wos-issues#163]
### Guidelines

- [X] I checked for duplicate suggestions
- [X] My suggestion intends to add or improve something

### Version

Development (Unstable)

### Topic

A game mechanic (e.g. a part, a feature, etc)

### Why are you making your suggestion?

Building the starter ship by hand could be considered quite tedious, especially for someone new to WoS, attempting to leave a starter planet. They may not fully understand the necessities for spaceflight and planet colonisation (e.g. Air Supply, StarMap, Telescope, Hyperdrive, etc.). Speaking from personal experience, there are little that are more frustrating than finally reaching space, then suffocating and having to start all over again.

### What do you propose? What should change, or what should be added?

I propose the addition of a starter ship template, with all the necessities built in. This would provide a list of parts that the players must gather to have a ship capable of colonising another planet. I feel that this addition would improve early-game UX, especially for new players.

**Comment by Erex147** [comment]
I agree with the above statement, prebuilt ruins the purpose. a list of required item and how to wire your ship makes it easier to know what you're doing could be better.

**Comment by idfreud** [comment]
Whether its a prebuilt ship model, a ship-building tutorial or a pondering system, I feel something must be added to WoS to let new players know how to lift off and colonise other planets. A new player shouldn't have to consult an online tutorial or an experienced player to properly play the game.

**Comment by mawesome4ever** [comment]
A better tutorial is required overall, but fixing current bugs is higher priority.
